content/zh/docs/ops/diagnostic-tools/cni/index.md

```diff
@@ -22,7 +22,7 @@ CNI 插件的默认日志级别是 `info`。要获得更详细的日志输出，
 `values.cni.logLevel` 安装选项并重新启动 CNI DaemonSet Pod 来更改级别。
 
 Istio CNI DaemonSet Pod 日志还提供了有关 CNI 插件安装的信息，和
-[race condition repairing](/zh/docs/setup/additional-setup/cni/#race-condition-mitigation)。
+[竞态条件修复](/zh/docs/setup/additional-setup/cni/#race-condition-mitigation)。
```

Comment:
```suggestion
[竞争条件和缓解措施](/zh/docs/setup/additional-setup/cni/#race-condition-mitigation)。
```
Click this url to learn details: https://istio.io/latest/zh/docs/setup/additional-setup/cni/#race-condition-mitigation